### PR TITLE
Contextual home

### PR DESCRIPTION
**Pull request by dylancom**
When you visit for e.g. https://lightning-roulette.com/ and there is an allowance already set, the extension will show the publisher/payments when opening the extension.

If not, it will show the default view.

### CODE REVIEW

**Comment by bumi** [comment]
Whoop whoop! super awesome!

File: src/app/screens/Home/index.jsx [hunk 1]
```diff
@@ -27,6 +30,17 @@ class Home extends React.Component {
   }
 
   componentDidMount() {
+    browser.tabs
+      .query({ active: true, lastFocusedWindow: true })
```
Comment (bumi) [inline src/app/screens/Home/index.jsx:34]: why not `currentWindow` do you know what's the difference?

Comment (dylancom) [reply src/app/screens/Home/index.jsx:34]: Good catch! Actually haven't tested with currentWindow instead.

Comment (dylancom) [reply src/app/screens/Home/index.jsx:34]: Pushed a change.
Should be currentWindow: https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab

**Pull request merged**